pages/agent/clusters.md

```diff
@@ -0,0 +1,97 @@
+# Clusters
+
+Clusters are a new way of managing your Buildkite agents. They allow teams to self-manage their agent pools, let admins create isolated sets of agents and pipelines within the one Buildkite organization, and help to make agents and queues more discoverable across your organization.
+
+The following diagram shows the architectural change when enabling clusters.
+
+<%= image "clusters-architecture.png", alt: "Diagram showing existing architecture and architecture with clusters" %>
+
+Clusters encapsulate groups of agents and pipelines. This enables the following:
+
+* Clusters are viewable to your entire organization, allowing engineers to better understand the agents and queues available for their pipelines.
+* Individual users or teams can maintain their own clusters. Cluster maintainers can manage queues and agent tokens and add and remove pipelines.
+* Pipelines can be assigned to a cluster, ensuring their builds run only on the agents connected to this cluster. These pipelines can also trigger builds only on other pipelines in the same cluster.
+
+## Enabling clusters
+
+Any Buildkite administrator can enable clusters for an organization. Once you enable clusters, you can only disable it by contacting support.
+
+To enable clusters:
+
+1. Navigate to your [organization’s pipeline settings](https://buildkite.com/organizations/~/pipeline-settings).
+1. In _Clusters_, select _Enable Clusters_.
+
+_Clusters_ will now appear in the global navigation.
+
+## Setting up a cluster
+
+To set up a new cluster:
+
+1. Navigate to _Clusters_
+2. Click _Create a Cluster_
+3. Enter a name, description and emoji
+4. Click _Create Cluster_
+
+### Setting up queues
+
+When you create your first cluster, it will have an initial “default” queue.
+
+To create additional queues:
+
+1. Navigate to the cluster’s _Queues_
+2. Click _Create a Queue_
+3. Enter a key and description
+4. Click _Create Queue_
+
+You should [supply the queue’s key][agent-queue-key] to the agents that you wish to run build jobs on the queue.
```

Comment:
Yeah, I agree this could do with a wholesale rethink :)

How about this? I started with your second suggestion, but moved agents to the start of the sentence to emphasise that the sentence itself is about setting up those agents.

> For agents to run jobs in a particular queue, [supply the queue’s key][agent-queue-key] to the agents.

Reply:
Nice! That's better